pkg/processor/trigger/kafka/test/kafka_test.go

```diff
@@ -717,6 +738,52 @@ func (suite *testSuite) resolveReceivedEventBodies(deployResult *platform.Create
 	return receivedBodies
 }
 
+func (suite *testSuite) ensureNumberOfCommittedOffsets(consumerGroup string, topic string, expectedNumberOfCommittedOffsets int) bool {
+	if topic == "" {
+		topic = suite.topic
+	}
+	numberOfCommittedOffsets, err := suite.getNumberOfCommittedOffsets(consumerGroup, topic, int(suite.NumPartitions))
+	if err != nil {
+		return false
+	}
+	return int(numberOfCommittedOffsets) == expectedNumberOfCommittedOffsets
+}
+
+func (suite *testSuite) getNumberOfCommittedOffsets(consumerGroup, topic string, partitions int) (int64, error) {
+	// Create an OffsetFetchRequest
+	request := &sarama.OffsetFetchRequest{
+		ConsumerGroup: consumerGroup,
+		Version:       1,
+	}
+
+	for partition := 0; partition < partitions; partition++ {
+		request.AddPartition(topic, int32(partition))
+	}
+
+	// Send the request to the broker
+	response, err := suite.broker.FetchOffset(request)
+	if err != nil {
+		return -1, fmt.Errorf("failed to fetch offsets: %w", err)
```

Comment:
```suggestion
		return -1, fmt.Errorf("Failed to fetch offsets: %w", err)
```

Just to make sure - does `fmt.Errorf` print the error even if it's not handled by the caller?

Reply:
@TomerShor if using this method directly, we would need to handle error - in this specific case we just ignore this error

Reply:
@TomerShor btw, that golang who fixes case:
![image](https://github.com/user-attachments/assets/82611fe9-7fa2-4179-ba64-eed9a91e0e71) because of this general rule (https://go.dev/wiki/CodeReviewComments#:~:text=Error%20strings%20should%20not%20be,something%20bad%22)%20not%20fmt.), however yes, in Nuclio we don differently

Reply:
In Nuclio we usually use `errors.Errorf` instead of `fmt` (although there are fmt usages here and there).

Reply:
@TomerShor Then I'll anyway need to wrap it with fmt.Sprintf().. and it will create to many braces in test code which is not even user facing, not sure it's worth it

Reply:
Agreed that it's not critical in the test, but for the sake of the discussion - 
`errors.Errorf` acts exactly the same as `fmt.Errorf`. It does not require `fmt.Sprintf`.
As opposed to `errors.New` which does require.
Here's a random example from our code:
https://github.com/nuclio/nuclio/blob/00f619aaa515b595ad30b09fde231aafdffa310d/pkg/platform/kube/functionres/lazy.go#L2917

Reply:
@TomerShor for the same of the discussion:

```
pkg/processor/trigger/kafka/test/kafka_test.go:768:14: printf: github.com/nuclio/errors.Errorf does not support error-wrapping directive %w (govet)
                return -1, errors.Errorf("Failed to fetch offsets: %w", err)

```